### PR TITLE
#781 Put dynawoexec in output directory instead of default directory.

### PR DESCRIPTION
**Pull request by gautierbureau**

### CODE REVIEW

File: dynawo/sources/Simulation/DYNSimulation.cpp [hunk 1]
```diff
@@ -753,12 +753,11 @@ void
 Simulation::simulate() {
   Timer timer("Simulation::simulate()");
 
-  string outputsDirectory = createAbsolutePath("outputs", context_->getInputDirectory());
-  if (!is_directory(outputsDirectory))
-    create_directory(outputsDirectory);
+  if (!is_directory(outputsDirectory_))
```
Comment (rosiereflo) [inline dynawo/sources/Simulation/DYNSimulation.cpp:756]: This will break the integration in tools (CVG to not name it) that use Dynawo.

Comment (gautierbureau) [reply dynawo/sources/Simulation/DYNSimulation.cpp:756]: Shit it is annoying to create this useless directory if we don't name the outputs directory this way...

Comment (rosiereflo) [reply dynawo/sources/Simulation/DYNSimulation.cpp:756]: Yes but this is where the tool gets this file, and it is hard coded.

Comment (gautierbureau) [reply dynawo/sources/Simulation/DYNSimulation.cpp:756]: ok

**Pull request closed without merging**